src/index.js

```diff
@@ -57,20 +58,23 @@ class DelegatedContentRouting {
    * Search the dht for providers of the given CID.
    *
    * - call `findProviders` on the delegated node.
-   * - does not support the `timeout` parameter, as this is specific to the delegate node.
    *
    * @param {CID} key
-   * @param {number} _timeout This is ignored and is only present to comply with the dht interface
+   * @param {number} timeout How long the query can take. Defaults to 30 seconds
```

Comment:
Since we are changing this, maybe we should change this param to an `options` object with the `timeout` property, and for now, keep accepting it as a number, but deprecate in the future, in favor of options.

Reply:
The major reason I didn't make the change to options was to comply with how the dht is doing things, https://github.com/libp2p/js-libp2p-kad-dht/blob/0a2f9fe418d364356d27152be6a7279ebb220d2c/src/index.js#L450. I'm good making the change but we should do that in the dht as well.

Reply:
You are right! I am just concerned that we may need to change all them in the future. I Can create a PR for `kad-dht` adding that behavior, what do you think?